android/app/build.gradle

```diff
@@ -23,7 +23,7 @@ if (flutterVersionName == null) {
 }
 
 apply plugin: 'com.android.application'
-if (System.getenv("IS_SIGNING_ENABLED") == "true") {
+if (System.getenv("IS_SIGNING_ENABLED") == "true" || System.getenv("IS_SIGNING_ENABLED") == null) {
```

Comment:
`IS_SIGNING_ENABLED` will be `null` when building locally. We must use firebase plugins when building locally, otherwise the app will break